content/docs/command-reference/root.md

```diff
@@ -10,11 +10,11 @@ usage: dvc root [-h] [-q | -v]
 
 ## Description
 
-This command returns the path to the root directory of the <abbr>DVC
-project</abbr>, relative to the current working directory. It can be used to
-build a path to a dependency, script, or <abbr>data artifact</abbr>, for
-example. Useful when working in a subdirectory of the project, and needing to
-refer to a file in another directory.
+Returns the path to the root directory of the <abbr>DVC project</abbr>, relative
```

Comment:
Should we also say "root directory" in the short intro (line 3) ?

Reply:
Agreed. Added this.

Reply:
Forgot to push?

Reply:
Was waiting for comments on other parts. Will push this by itself for now.